devices: # the names here are used for publisher's namespace
  - stereo
  - stereo2
  - stereo3
  - stereo4
  - stereo5
  # - stereo11
  - fisheye

stereo: 
  type: d400 # for loading type specific yaml
  sn: '843112073923' # D435i
  stereo: true
  depth: false
  poseimu: true
  auto_exposure_mode: manual # internal, custom, or manual
  hardware_synchronisation_mode: none

stereo2: 
  type: d400 # for loading type specific yaml
  sn: '843112072452' # D435i
  stereo: false
  depth: true
  color: false
  poseimu: false
  auto_exposure_mode: manual
  hardware_synchronisation_mode: master

stereo3: 
  type: d400 # for loading type specific yaml
  sn: '912112071531' # D435i
  stereo: false
  color: false
  depth: true
  poseimu: false
  auto_exposure_mode: manual
  hardware_synchronisation_mode: slave

stereo4: 
  type: d400 # for loading type specific yaml
  sn: '912112071518' # D435i
  stereo: false
  depth: true
  poseimu: false
  auto_exposure_mode: manual
  hardware_synchronisation_mode: slave

stereo5: 
  type: d400 # for loading type specific yaml
  sn: '912112073894' # D435i
  stereo: false
  depth: true
  poseimu: false
  auto_exposure_mode: manual
  hardware_synchronisation_mode: slave

stereo11:
  type: d400 # for loading type specific yaml
  sn: '746112061685' # D415
  stereo: true
  depth: false
  poseimu: false
  auto_exposure_mode: manual
  hardware_synchronisation_mode: slave

fisheye: 
  type: t265
  sn: '908412110033' # T265
  stereo: true
  depth: false
  poseimu: true
  auto_exposure_mode: manual # custom